### PR TITLE
1D stock market support (implements #1592)

### PR DESCRIPTION
**Pull request by dfannius**

### CODE REVIEW

File: assets/app/view/game/stock_market.rb [hunk 1]
```diff
@@ -22,47 +22,84 @@ class StockMarket < Snabberb::Component
...
+          textAlign: 'center',
+          lineHeight: '0',
+        }
+
+        price_style = {
+          fontSize: '100%',
+          textAlign: 'center',
+        }
+
+        max_num_corps = @game.stock_market.market.first.map { |p| p.corporations.size }.push(MIN_NUM_TOKENS).max
```
Comment (tobymao) [inline assets/app/view/game/stock_market.rb:77]: [@game.stock_market.markets[0].map { |p| p.corporations.size }.max, MIN_NUM_TOKENS].max

i found this a bit more readable

Comment (tobymao) [reply assets/app/view/game/stock_market.rb:77]: i'll leave it up to you

**Review by tobymao — APPROVED** [review]

**Comment by tobymao** [comment]
feel free to merge whenever you'd like

File: assets/app/view/game/stock_market.rb (hunk 1 above)
Comment (dfannius) [reply assets/app/view/game/stock_market.rb:77]: If you're leaving it up to me, I'll keep it as it is, to have one `max` rather than two.

**Pull request merged**